src/Modules/ElementUtils/inheritance.js

```diff
@@ -6,13 +6,13 @@ const INHERITED_MEMBER_QUERY = {
   $or: [
     {
       'meta.links': {
-        href: 'http://example.com/inherited-member/',
+        href: 'http://refract.link/inherited-member/',
```

Comment:
@JackuB Could you please elaborate why is this change needed to fix referencing issue?